interface/super/rules/controllers/browse/view/list.php

```diff
@@ -6,8 +6,8 @@
  // as published by the Free Software Foundation; either version 2
  // of the License, or (at your option) any later version.
 
- require_once("../../../../../globals");
- require_once("$srcdir/headers.inc.php");
+ require_once(dirname(__FILE__)."/../../../../../globals.php");
+ require_once($GLOBALS['srcdir']."/headers.inc.php");
```

Comment:
@punwai i guess the global.php file was not required. thanks for fixing the link for that. Also i think it is not necessary to do `$GLOBALS['srcdir']` cause globals is already required 😃 . Try using the `$srcdir` and lets see what it gives